.travis_publish

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+# save current path
+REPO_PATH=$PWD
+
+# Save some useful information
+SHA=`git rev-parse --verify HEAD`
+
+# Clone the existing gh-pages for this repo into $HOME/gh-pages/
+# Create a new empty branch if gh-pages doesn't exist yet (should only happen on first deply)
```

Comment:
typo deply -> deploy

Reply:
this is just a comment, but I will fix it